lib/AST/ModuleDependencies.cpp

```diff
@@ -584,6 +584,14 @@ void SwiftDependencyTracker::addCommonSearchPathDeps(
       recordFiles(val.SearchPath);
       break;
     }
+
+    // '-load-plugin <library path>:<server path>#<module name>,...'.
+    case PluginSearchOption::Kind::LoadPlugin: {
+      auto &val = entry.get<PluginSearchOption::LoadPlugin>();
+      FS->status(val.LibraryPath);
+      // TODO: what else do we need?
```

Comment:
not familiar with this part of the codebase — should I be recording `val.ServerPath` as well?

Reply:
Yes, I think we need to record `.wasm` files under the search path